app/assets/stylesheets/application.scss

```diff
@@ -277,6 +276,18 @@
     width: 60px;
   }
 
+//panel elements
+  .field-log-in {
+    color: #ead61c;
+  }
+
+  .panel-reg {
+    background-color: rgba(0,0,0,0.25);
+    margin-top: 20px;
```

Comment:
Properties should be ordered background-color, margin-left, margin-right, margin-top